scripts/confidence_ensembles/test_confidence_ensembles.py

```diff
@@ -37,9 +37,23 @@
         (
             "ensemble.0.model=stt_es_fastconformer_hybrid_large_pc "
             "ensemble.1.model=stt_it_fastconformer_hybrid_large_pc "
-            "transcription.decoder_type=ctc"
+            "transcription.decoder_type=ctc "
```

Comment:
Space ?

Reply:
Feels a bit safer to end with space this kind of multiline string specification as it's easy to forget to add it when new parameters are added. Similar to how we end multiline dictionaries with commas.